test/blackbox-tests/test-cases/pkg/pin-stanza/build-command-dune-project.t

```diff
@@ -51,11 +51,11 @@ Demonstrate the build command we construct for different types of projects:
   - mixed.dev
   - template.dev
   $ build_command() {
-  > grep "(dune)" dune.lock/$1.pkg
+  > grep "$1" dune.lock/$2.pkg
   > }
-  $ build_command dune-only
+  $ build_command "(dune)" dune-only
   (dune)
-  $ build_command mixed
-  (dune)
-  $ build_command template
+  $ build_command "(dune)" template
   (dune)
+  $ build_command "(build" mixed
+  (build
```

Comment:
This is good but I think it would be also useful to show that pinning and building a project that doesn't use dune works.